src/Raven.Server/Smuggler/Documents/Handlers/SmugglerHandler.cs

```diff
@@ -277,6 +279,30 @@ public Task GetImport()
             }
         }
 
+        [RavenAction("/databases/*/smuggler/migrate", "POST", AuthorizationStatus.ValidUser)]
```

Comment:
This should be done only by `DbAdmin`, not `ValidUser`. The reason is that we are using the _server_ credentials to do stuff, which we might not want them to use.

Reply:
done